infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -31,12 +32,20 @@
 
     private final ServerContext serverContext;
 
+
     /**
      * Interval in checking presence of management layout to start management agent tasks.
-     * In milliseconds.
+     * In seconds.
```

Comment:
it doesn't really matter if it in seconds or millis, the variable has `Duration` type. Maybe we don't need to explain this in javadoc?

Reply:
Just to make them consistent.

Reply:
just delete it. It will make comments very consistent)

Reply:
done.